changelog.d/6318.bugfix

```diff
@@ -0,0 +1 @@
+Fix loop in timeline and simplify management of chunks and timeline events.
```

Comment:
Does this also close the following? They seem like duplicates of https://github.com/vector-im/element-android/issues/5166 anway

 - https://github.com/vector-im/element-android/issues/5690
 - https://github.com/vector-im/element-android/issues/6342